arch/arm64/boot/dts/freescale/crocodile.dts

```diff
@@ -181,6 +181,17 @@
 		model = "imx-audio-micfil";
 		cpu-dai = <&micfil>;
 	};
+	bat: battery {
```

Comment:
Not sure if we need a label for the battery node, even if it is listed in the documentation as an example...

Can you please use the simple naming scheme here for the node, like:

```
internal-battery {
	<content>
}
```

Reply:
@zandrey ,the lable was used by node "bq78z100: fuel-gauge@55" maybe we need it.